solarforecastarbiter/io/reference_observations/srml_reference_sites.json

```diff
@@ -307,7 +307,7 @@
                 "site_id": "",
                 "timezone": "Etc/GMT+8"
             },
-            "uncertainty": 0,
+            "uncertainty": null,
```

Comment:
do we need a database migration?

Reply:
good question. This was just to make that one observation consistent with all the rest, but I think they all currently have uncertainty==0 in prod right now, as do basically all of the other reference observations. We probably don't want them to be null but need to think about what a reasonable number for at least each network/variable